packages/patternfly-4/react-core/src/components/AboutModal/examples/SimpleAboutModal.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { AboutModal, Button } from '@patternfly/react-core';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/patternfly-next/components/Content/styles.css';
+import brandImg from './pf_mini_logo_white.svg';
+import logoImg from './pf_logo.svg';
+import heroImg from './pfbg_992.jpg';
```

Comment:
If you add jpg to react-docs/gatsby-node.js (look for `/(png|svg)/` then this should work

Reply:
done